connect_box/__init__.py

```diff
@@ -74,7 +108,65 @@ def __init__(
             self.token = None
             raise exceptions.ConnectBoxNoDataAvailable() from None
 
-        return self.devices
+    async def async_get_downstream(self):
+        """Get the current downstream cable modem state."""
+
+        if self.token is None:
+            await self.async_initialize_token()
+
+        raw = await self._async_ws_function(CMD_DOWNSTREAM)
+
+        try:
+            xml_root = element_tree.fromstring(raw)
+            self.ds_channels.clear()
+            for ds in xml_root.iter('downstream'):
+                self.ds_channels.append(DownstreamChannel(
+                    int(ds.find('freq').text),
+                    int(ds.find('pow').text),
+                    ds.find('mod').text,
+                    ds.find('chid').text,
+                    float(ds.find('RxMER').text),
+                    int(ds.find('PreRs').text),
+                    int(ds.find('PostRs').text),
+                    ds.find('IsQamLocked').text == '1',
+                    ds.find('IsFECLocked').text == '1',
+                    ds.find('IsMpegLocked').text == '1'
```

Comment:
use type convert function from attrs

Reply:
Sorry, but I don't want to read up on attrs any further. If this blocks the PR from being merged, please let me know, then I'll close it.

Reply:
Okay, but please run tox and fix the issue like the attribute name are not python PEP legal.

Reply:
I fixed the issues tox reported (both style and formatting) and it's not reporting any more issues for me now, although weirdly enough the property names are certainly NOT snake case. Is this check really working correctly?